hack/cleanup/konnect_runtime_groups.go

```diff
@@ -4,42 +4,37 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io"
 	"net/http"
 	"time"
 
 	"github.com/go-logr/logr"
-	"github.com/oapi-codegen/runtime/types"
 	"github.com/samber/lo"
 
-	cp "github.com/kong/kubernetes-ingress-controller/v3/internal/konnect/controlplanes"
+	sdkkonnectgo "github.com/Kong/sdk-konnect-go"
```

Comment:
Should the packages from `sdk-konnect-go` have a separate section in imports?

Reply:
IMHO yes, that's why I made it this way here and added a rule in the linter: https://github.com/Kong/kubernetes-ingress-controller/blob/faedca400fe43f6dce48540e5359f4fbee475a7b/.golangci.yaml#L57.

WDYT?